src/assets/sampleBobaAPI.tsx

```diff
@@ -164,13 +165,13 @@ export const bobaList: BobaProps[] = [
     },
 ];
 
-export const milkList: MilkListProps = {
-    'milk-1': { name: 'Whole Milk' },
-    'milk-2': { name: '2% Milk' },
-    'milk-3': { name: 'Almond Milk' },
-    'milk-4': { name: 'Soy Milk' },
-    'milk-5': { name: 'Oat Milk' },
-};
+export const milkList: MilkProps[] = [
+    { uuid: 'b19dd65c-e087-4925-b041-35c4f45db7b3', name: 'Whole Milk' },
+    { uuid: '17a9b305-a9b6-49ea-afb9-0c0f4fa3d24e', name: '2% Milk' },
+    { uuid: 'd3b993b0-41ed-4ea7-9348-f1c9b31308f9', name: 'Almond Milk' },
+    { uuid: '081c423a-8348-4886-ad5f-1237b3b44487', name: 'Soy Milk' },
+    { uuid: '7077e01b-3aa8-4fd0-9e0f-3fc0e1fb5b03', name: 'Oat Milk' },
+];
```

Comment:
Remove this duplicated list of boabList, milkList, toppingList from sampleAPI and read from one file in backend/database.
There should only be TS props/interface at front end

Reply:
not familer with reading file from backend in frontend. within the backend , i use 
```
with open('./database/data.json') as f:
        bobajson = json.load(f)['bobaData']
```
      was it be same case

Reply:
actually the code is for Python and its use for backend so it will not works

Reply:
use require/fetch as discussed

Reply:
i tried multiple way of fetching with json file and didnt work so only made the changes on the backend and everything remain the same